src/Twilio/Rest/Accounts/V1/Credential/PublicKeyResource.cs

```diff
@@ -154,7 +155,8 @@ private static Request BuildCreateRequest(CreatePublicKeyOptions options, ITwili
                 HttpMethod.Post,
                 Rest.Domain.Accounts,
                 "/v1/Credentials/PublicKeys",
-                postParams: options.GetParams()
+                postParams: options.GetParams(),
+                headerParams: null
```

Comment:
Can these be reverted?

Reply:
Absolutely. Just need to add some additional logic